src/test-reader/mocha-reader/index.js

```diff
@@ -95,6 +99,94 @@ function applyOnly(rootSuite, eventBus) {
     });
 }
 
+function addLocationToRunnables(inBus, config, runnableOpts) {
+    if (!runnableOpts || !runnableOpts.saveLocations) {
+        return;
+    }
+
+    const sourceMapSupport = tryToRequireSourceMapSupport();
+    const { suiteMethods, testMethods } = getMethodsByInterface(config.ui);
+
+    inBus.on(MochaEventBus.events.EVENT_FILE_PRE_REQUIRE, ctx => {
```

Comment:
Here ctx - it is global value, which has `describe`, `it` and others for `bdd` interface for example